lib/openstudio-standards/standards/Standards.Model.rb

```diff
@@ -14,229 +14,514 @@ def define_space_multiplier
+      end
 
-    # Fix EMS references.
-    # Temporary workaround for OS issue #2598
-    model_temp_fix_ems_references(model)
+      # Set the baseline fan power for all zone HVAC
+      model.getZoneHVACComponents.sort.each do |zone_hvac|
+        zone_hvac_component_apply_prm_baseline_fan_power(zone_hvac)
+      end
 
-    # Delete all the unused resource objects
-    model_remove_unused_resource_objects(model)
+      # Set the baseline number of boilers and chillers
+      model.getPlantLoops.sort.each do |plant_loop|
+        # Skip the SWH loops
+        next if plant_loop_swh_loop?(plant_loop)
 
-    # @todo turn off self shading
-    # Set Solar Distribution to MinimalShadowing... problem is when you also have detached shading such as surrounding buildings etc
-    # It won't be taken into account, while it should: only self shading from the building itself should be turned off but to my knowledge there isn't a way to do this in E+
+        plant_loop_apply_prm_number_of_boilers(plant_loop)
+        plant_loop_apply_prm_number_of_chillers(plant_loop, sizing_run_dir)
+      end
 
-    model_status = 'final'
-    model.save(OpenStudio::Path.new("#{sizing_run_dir}/#{model_status}.osm"), true)
+      # Set the baseline number of cooling towers
+      # Must be done after all chillers are added
+      model.getPlantLoops.sort.each do |plant_loop|
+        # Skip the SWH loops
+        next if plant_loop_swh_loop?(plant_loop)
 
-    # Translate to IDF and save for debugging
-    forward_translator = OpenStudio::EnergyPlus::ForwardTranslator.new
-    idf = forward_translator.translateModel(model)
-    idf_path = OpenStudio::Path.new("#{sizing_run_dir}/#{model_status}.idf")
-    idf.save(idf_path, true)
+        plant_loop_apply_prm_number_of_cooling_towers(plant_loop)
+      end
+
+      # Run sizing run with the new chillers, boilers, and cooling towers to determine capacities
+      if model_run_sizing_run(model, "#{sizing_run_dir}/SR2") == false
+        return false
+      end
+
+      # Set the pumping control strategy and power
+      # Must be done after sizing components
+      model.getPlantLoops.sort.each do |plant_loop|
+        # Skip the SWH loops
+        next if plant_loop_swh_loop?(plant_loop)
+
+        plant_loop_apply_prm_baseline_pump_power(plant_loop)
+        plant_loop_apply_prm_baseline_pumping_type(plant_loop)
+      end
+
+      OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.Model', '*** Applying Prescriptive HVAC Controls and Equipment Efficiencies ***')
+
+      # Apply the HVAC efficiency standard
+      model_apply_hvac_efficiency_standard(model, climate_zone)
+
+      # Set baseline DCV system
+      model_set_baseline_demand_control_ventilation(model, climate_zone)
+
+      # Final sizing run and adjustements to values that need refinement
+      model_refine_size_dependent_values(model, sizing_run_dir)
+
+      # Fix EMS references.
+      # Temporary workaround for OS issue #2598
```

Comment:
looks like this was addressed https://github.com/NREL/OpenStudio/issues/2598

Reply:
@mdahlhausen As I read the comments in this issue report, it looks like it was not fixed, even though it was closed:

[brianlball](https://github.com/brianlball) commented [on May 8, 2017](https://github.com/NREL/OpenStudio/issues/2598#issuecomment-299889432)
"This was a design decision to leave that field as just an /alpha and it doesnt not do any UID substitution since that field can be really anything. Its similiar to the Actuator Component type and Control Type fields or the Sensor Index Key Name. Its just too hard to figure out what the "right" values are so its up to the user to put in the right /alpha.

The solution is as you said, the end user should use "name" instead of "handle" in their measure"